hibike/utils/device_names.sh

```diff
@@ -0,0 +1,10 @@
+#/bin/bash
+
+#returns a list of all device names.  Used in combination with other linux hackery for tab-completion
+
+
+#this function is not meant to be called from the "utils" folder.  It will fail because of pathing problems.
+#it is meant to be called from the "hibike" folder, since that is where flash.sh is located.
+
+
+ls devices/
```

Comment:
Could you add a newline at the end of the file?

(See how there's a red icon when github displays this line? That means there's no newline at eof. It's standard practice to end a file in a newline, because of how lack of a newline interacts with diffing tools)